playbooks/simple/filesystem_delete.yml

```diff
@@ -0,0 +1,15 @@
+---
+- hosts: localhost
+  vars_files:
+    - ../../group_vars/all
+  tasks:
+    - name: Delete filesystem
+      fileInterface:
+        login:
+          unityIP: "{{ unity_host }}"
```

Comment:
In #41 and #44 we remove unity_host group var. Let's change it to unity_ip_source or unity_ip_dest: